contracts/Bridge.sol

```diff
@@ -1,18 +1,18 @@
 //SPDX-License-Identifier: UNLICENSED
 pragma solidity ^0.8.0;
 
-import "./IERC20Mintable.sol";
+import { WrappedToken } from "./WrappedToken.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
 
-contract Bridge {
+contract Bridge is Ownable{
```

Comment:
It is considered an anti-pattern to make the bridge ownable as it prevents others from using it. The users don't have to specifically consume it though it's (your) front end, they can call it however they like.